components/shell/CommandPalette/actions.ts

```diff
@@ -25,9 +16,10 @@ export const getStaticActions = ({
     shortcut: ['w'],
     keywords: 'workspace,panel',
     perform: () => {
-      const newParams = new URLSearchParams(searchParams)
+      const pathname = window.location.pathname
```

Comment:
this has been causing a bunch of flaky test failures for a while now because of how nextjs was keeping cached version of pathname.